scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -1429,6 +1431,65 @@ while true; do
     fi
   fi
 
+      # Mithril Signer Section
+      SERVICE_EXISTS=$(systemctl list-units --type=service --all | grep -q 'cnode-mithril-signer.service'; echo $?)
+      SIGNER_SERVICE_STATUS=$(systemctl status cnode-mithril-signer.service 2>/dev/null | grep 'Active:' | awk '{print $3}' | sed 's/[()]//g')
+      MITHRIL_SIGNER_METRICS=$(curl -s http://127.0.0.1:9090/metrics 2>/dev/null | grep -v -E "HELP|TYPE" | sed 's/mithril_signer_//g')
+
+      if [[ "$nodemode" = "Core" ]] && [[ "$SERVICE_EXISTS" -eq 0 ]] && [[ "$SIGNER_SERVICE_STATUS" == "running" ]]; then
+
+        printf "${mithrildivider}\n" && ((line++))
+
+        get_metric_value() {
+            local metric_name="$1"
+            echo "$MITHRIL_SIGNER_METRICS" | awk -v metric="$metric_name" '{for(i=1;i<=NF;i++){if($i==metric){print $(i+1)}}}'
```

Comment:
Though awk is fairly quick, using bash internal REMATCH is even faster. For example on how this is done, check getNodeMetrics() in env file under else section. It has example for parsing prometheus output from node. 

Reason I mention this is because due to the frequent refreshes, any millisecond saved in processing, is good. 

Something else to consider, as these metrics are fairly static, would be to not re-fetch metrics on every 2s refresh, but wait for x number of refreshes between updates.